metricbeat/module/redis/info/_meta/fields.yml

```diff
@@ -73,110 +67,219 @@
           type: long
           description: >
           format: bytes
-            Used memory.
-
+            Total number of bytes allocated by Redis.
         - name: used.rss
           type: long
           format: bytes
           description: >
-            Used memory rss.
-
+            Number of bytes that Redis allocated as seen by the operating system (a.k.a resident set size).
         - name: used.peak
           type: long
           format: bytes
           description: >
-            Used memory peak.
-
+            Peak memory consumed by Redis.
         - name: used.lua
           type: long
           format: bytes
           description: >
-            Used memory lua.
+            Used memory by the Lua engine.        
+        - name: used.dataset
+          type: long
+          format: bytes
+          description: >
+            The size in bytes of the dataset 
 
         - name: max.value
           type: long
           format: bytes
           description: >
             Memory limit.
-
         - name: max.policy
           type: keyword
-          description:
+          description: >
             Eviction policy to use when memory limit is reached.
 
+        - name: fragmentation.ratio
+          type: float
+          description: >
+            Ratio between used_memory_rss and used_memory
+        - name: fragmentation.bytes
+          type: bytes
```

Comment:
@a3dho3yn I have run the failing test locally, and it seems that the failure is caused by this, `bytes` can be used as `format`, but not as `type`.